README.adoc

```diff
@@ -566,7 +569,7 @@ include::finish/deploy.yaml[]
 ----
 
 The `inventory` microservice displays the `hostname` and `systemLoad` of all `system` microservices that are publishing
-messages to the Kafka broker. Since only one `system` pod is running, there is only one element in the `inventory`.
+messages to the Kafka broker. Because only one `system` pod is running, there is only one element in the `inventory`.
```

Comment:
suggested edit
Because only one `system` pod is running, there is only one element in the `inventory`.
--->
Because only one `system` pod is running, only one element displays in the `inventory` microservice.